src/biogeochem/CNPhenologyMod.F90

```diff
@@ -838,7 +854,7 @@ subroutine CNSeasonDecidPhenology (num_soilp, filter_soilp       , &
 
             ! test for switching from dormant period to growth period
             if (dormant_flag(p) == 1.0_r8) then
-
+               onset_thresh = 0.0_r8
```

Comment:
This onset_thresh doesn't need to be a real variable, it should just be a logical.

Reply:
I'll file this as an issue and do this in a bit-for-bit refactoring later. Simulations are being done with the current formulation so I need to make sure I don't mess anything up.

Reply:
Filed this as issue #1346